docs/build_docs.sh

```diff
@@ -3,7 +3,17 @@
 
 #!/bin/sh
 echo "WARNING: This script has been deprecated. Please start calling build_docs.py instead"
-python3 $GEOIPS_PACKAGES_DIR/geoips/docs/build_docs.py --license-url $GEOIPS_REPO_URL --geoips-docs-path $GEOIPS_PACKAGES_DIR/$2/docs --output-dir "$GEOIPS_PACKAGES_DIR/$2/docs/build/sphinx/html" --repo-path $1 $2
-
-
+reponame=$2
+repopath=$1
+save_temp_dir=$GEOIPS_OUTDIRS/scratch/docs/$reponame
+python3 $GEOIPS_PACKAGES_DIR/geoips/docs/build_docs.py \
+  --license-url $GEOIPS_REPO_URL \
+  --geoips-docs-path $GEOIPS_PACKAGES_DIR/geoips/docs/ \
+  --output-dir "$GEOIPS_PACKAGES_DIR/$2/docs/build/sphinx/html" \
+  --save-temp-dir "$save_temp_dir" \
+  --force FORCE \
+  --repo-path $repopath \
+  $reponame
 
+echo "Temporary files written to:"
+echo $save_temp_dir
```

Comment:
We may want to rename this "test_build_docs.sh" rather than fully deprecating, we'll decide with final docs update from integration branch.

Reply:
I'm going to just resolve, this will come up with the current doc PRs